src/main/java/org/mafagafogigante/dungeon/commands/CommandDescription.java

```diff
@@ -20,7 +20,7 @@
    */
   public CommandDescription(String name, String info) {
     if (name == null) {
-      throw new IllegalArgumentException("Cannot create CommandDescription with null name.");
+      throw new IllegalArgumentException("Cannot create CommandDescription with null name");
```

Comment:
We should get rid of the check and use a `@NotNull` annotation instead.

Back when I wrote this I was not using the annotation.